site/apache.md

```diff
@@ -62,7 +62,9 @@ loadBalancerAppPrepath: /weblogic
 
 ```
 
-Users can then access an application from outside of the Kubernetes cluster by using `http://<host>:30305/weblogic/<application-url>,` and the administrator can access the Administration Console by using `http://<host>:30305/console`.
+It is sometimes, but rarely, desirable to expose a WebLogic administrative host and port through a load balancer to a public network.  If this is needed, then, once the `weblogic-domain-apache.yaml` file is generated, you can customize exposure of the WebLogic admin admin host and port by uncommenting the `WEBLOGIC_HOST` and `WEBLOGIC_PORT` environment variables in the file.   If this files' resources have already been deployed (as happens automatically when running `create-weblogic-domain.sh`), one way to make the change is to delete the files' running Kubernetes resources via `kubectl delete -f weblogic-domain-apache.yaml`, and then deploy them again via `kubectl create -f weblogic-domain-apache.yaml`.
```

Comment:
"If this is needed, then, once" should be "If this is needed, then, after the ..."

Reply:
Typo, "WebLogic admin admin host"

Reply:
"If this files' resources..." I think should be "If this file's resources..." (it seems to me that you are referring to one file's resources).